src/BotiumBotmockExporter.ts

```diff
@@ -0,0 +1,443 @@
+import "dotenv/config";
+import {BaseExporter, Resources, FileWriter, Kind, ProjectReference} from "@botmock/export";
+import {Botmock, DataTransformation, KeyDependent} from "@botmock/export/src/types/index";
+import {BotDriver} from "botium-core"
+import {sanitize} from "sanitize-filename-ts";
+
+const SCRIPTING_FORMAT_TXT = 'SCRIPTING_FORMAT_TXT'
+const SCRIPTING_TYPE_CONVO = 'SCRIPTING_TYPE_CONVO'
+const SCRIPTING_TYPE_UTTERANCES = 'SCRIPTING_TYPE_UTTERANCES'
+
+import {Botium} from "./types";
+
+interface Log {
```

Comment:
In my opinion these interfaces could be moved into the types as well.

Reply:
Those are internal interfaces. types/index.ts are external. Is this not correct to separate those this way?

Reply:
I saw this kind of code organization in typescript, that the types were defined in a separate file. But it's absolutely not critical.